qa/os/src/test/java/org/elasticsearch/packaging/test/KeystoreManagementTests.java

```diff
@@ -58,6 +58,7 @@
     public static final String ERROR_KEYSTORE_NOT_FOUND = "ERROR: Elasticsearch keystore not found";
     private static final String USERNAME = "elastic";
     private static final String PASSWORD = "nothunter2";
+    private static final String KEYSTORE_PASSWORD = "keystore-password";
```

Comment:
I am mostly re-ordering tests in this class so that we dont need to remove and re-create the keystore. The reason is that the keystore contains passwords for TLS keystores and if we remove and recreate it, elasticsearch can't start